docs/src/02-learn/tutorials/1-scaffolding/1-project-init/index.md

```diff
@@ -22,23 +22,25 @@ Open a PowerShell terminal and navigate to your new project folder.
 
 Run the following commands to check your current versions:
 
-    ```powershell
-    # Check Node.js version
-    node -v
+```pwsh
+# Check Node.js version
+node -v
 
-    #Check NPM version
-    npm -v
-    ```
+#Check NPM version
+npm -v
+```
 
 Validate that their versions match the compatibility list stated in the [installation guide](../../../../01-install/index.md).
 If needed, use `nvm` command to fix it.
 
 ## 4. Initialize Project
 
-You can initialize a MES Customization or App project workspace using the `cmf init` command.  The [Project Types Concept](../../../concepts/project-types/index.md) page provides details on this command dependency versions. The following example demonstrates its usage.
+You can initialize a MES Customization or App project workspace using the `cmf init` command. The following examples illustrate its usage.
 
-!!! note "MES ISO location usage"
-    You only need to provide a MES ISO location if you are running your MES on Windows or one of its optional components (more details on [MES System Architecture][help-MES-architecture] page).
+!!! note "Software and MES ISO Dependencies"
+
+    * **Dependencies Version**: To determine the software dependencies version to use on the `init` command, check the instructions on the [Project Types Concept](../../../concepts/project-types/index.md) page.
+    * **MES ISO location is optional**: You only need to provide this parameter if your MES or one of its optional components **runs** on a Windows environment. More details on MES components are available on the [MES System Architecture][help-MES-architecture] page.

```

Comment:
please open an issue for this, as this isn't handled correctly by the CLI now. We expect the ISO location anyway. If you don't specify it, a placeholder is left instead of the path.

Reply:
#496 will address this